draft-ietf-rats-architecture.md

```diff
@@ -828,12 +828,12 @@ In some scenarios, Evidence might contain sensitive information such as
 Personally Identifiable Information.
 Thus, an Attester must trust entities to which it conveys Evidence, to not
 reveal sensitive data to unauthorized parties.
-The Verifier might share this information with other authorized parties, according to rules that it controls.
+The Verifier might share this information with other authorized parties, according to some predefined rules.
 In the background-check model, this Evidence may also be revealed to Relying Party(s).
 
-In some cases where Evidence contains sensitive information, an Attester
-might even require that a Verifier first go through a TLS authentication or a remote attestation procedure with it before the Attester
-will send the sensitive Evidence.  This can be done by having the
+When Evidence contains sensitive information, an Attester
+typically requires that a Verifier authenticates itself (e.g., at TLS session establishment) and might even request a remote attestation before the Attester
+sends the sensitive Evidence.  This can be done by having the
```

Comment:
Text is improved, but I am not really happy with the text in general, as of yet.